### PR TITLE
TBS OKTA-453254 Updates UI references in styling guides

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Removes paths that refer to **Settings** > **Customization**, etc.
- **Is this PR related to a Monolith release?** No.

### Resolves:
* [OKTA-453254](https://oktainc.atlassian.net/browse/OKTA-453254)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ You can add any HTML, CSS, or JavaScript to the sign-in page and also customize
 
 ### Edit the sign-in page
 
-The **Custom Sign-In Page** offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, in the Admin Console, go to **Settings** > **Customization**. On the **Customization Page**, click the **Custom Sign in** tab.
+The **Custom Sign-In Page** offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, in the Admin Console, go to **Customizations** and then **Sign-in page code editor**.
 
-> **Note**: In Okta Identity Engine, select **Customizations** and then **Sign-in page code editor** to find the Sign-In Widget editor.
+> **Note**: In Okta Classic Engine, select **Settings** > **Customization**. On the **Customization Page**, click the **Custom Sign in** tab.
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:48]: should we mention that this page is view only until a custom domain is successfully configured?

**Review by albertchen-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:48]: As discussed, we do just above this section. But I agree that it's important enough to repease.

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md [hunk 2]
```diff
@@ -43,9 +43,14 @@ You can add any HTML, CSS, or JavaScript to the sign-in page and also customize
 
 ### Edit the sign-in page
 
-The **Custom Sign-In Page** offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, in the Admin Console, go to **Settings** > **Customization**. On the **Customization Page**, click the **Custom Sign in** tab.
+The **Custom Sign-In Page** offers both basic and advanced customization options to create a completely transformed sign-in experience.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:46]: remove "both"

File: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ The [Brands API](/docs/reference/api/brands/) allows you to set all of the follo
 
 ## Important: Overriding themes in the code template editors
 
-You can customize individual parts of your Okta org experience using the various code editors available in the Admin Console (for example, **Settings** > **Customizations** > **Custom Sign-in** to customize the Okta-hosted sign-in page). However, if you use the Brands API to delete or augment any of the [branding variables](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-variables), the API calls override any changes you make using the Admin Console.
+You can customize individual parts of your Okta org experience using the various code editors available in the Admin Console (for example, **Customizations** > **Sign-in page code editor** to customize the Okta-hosted sign-in page). However, if you use the Brands API to delete or augment any of the [branding variables](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-variables), the API calls override any changes you make using the Admin Console.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-themes/index.md:40]: Remove "of the" in " .. any of the branding variables .." so it becomes "any branding variables"

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
two small nits.

File: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-themes/index.md:40]: I've not seen the use of "Important" as part of a heading before, although maybe it's useful to highlight. Perhaps ping Style channels on this. I don't want to hold up PR on this item, though.

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/customize-themes/index.md:40]: @brianduffield-okta I'll just remove it. It's legacy content so I'm not too attached to it :)

File: packages/@okta/vuepress-site/docs/guides/customize-themes/index.md [hunk 4]
```diff
@@ -35,9 +35,9 @@ The [Brands API](/docs/reference/api/brands/) allows you to set all of the follo
 * Favicon
 * Background image
 
-## Important: Overriding themes in the code template editors
+## Overriding themes in the code template editors
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/customize-themes/index.md:38]: avoid using -ING verbs in titles and headings. 
```suggestion
## Override themes in the code template editors
```

**Review by susanharper-okta — APPROVED** [review]
one comment - approving to not hold you up

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

**Pull request merged**